libvirt.go

```diff
@@ -318,6 +320,44 @@ const (
 	RebootParavirt
 )
 
+// DomainMemoryStatTag specifies domain memory tags
+type DomainMemoryStatTag uint32
+
+const (
+	// DomainMemoryStatTagSwapIn - The total amount of data read from swap space (in kB).
+	DomainMemoryStatTagSwapIn DomainMemoryStatTag = 0 << iota
+	// DomainMemoryStatTagSwapOut - The total amount of memory written out to swap space (in kB).
+	DomainMemoryStatTagSwapOut
```

Comment:
nitpick: let's add a `\n` after each const definition here to match the rest of the file.